packs/src/main/AndroidManifest.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-          package="@CONFIG.APP_PACKAGE_NAME@"
```

Comment:
Right, but did you confirm you can build the TextIt branded version as well? We can't break that workflow.